.circleci/config.yml

```diff
@@ -15,13 +15,19 @@ parameters:
   generate_screenshots:
     type: boolean
     default: false
+  beta_build:
+    type: boolean
+    default: false
+  release_build:
+    type: boolean
+    default: false
```

Comment:
Maybe I missed something, but it seems to me that those are _always_ OR'd at call site, so wondering if we need 2 separate parameters insted of a single one?

 - Every workflow that runs when non is set is using `unless: { or: … }` containing both `beta_build` and `release_build` in the `or`
 - The "Release Build" workflow runs if either of those is set (`when: { or: … }` with both those params)
 - I haven't seen anywhere else in the config (scripts param substitutions, etc)

So maybe unifying them under a single one could simplify the config?

Reply:
Yeah, right! I initially wanted to keep it to align the infrastructure with the iOS one, but the rest of the flow is different enough that this actually doesn't make sense as we can discriminate beta builds in a safer way on Android.
Updated in 40c94d0.